servicefabric/2016-01-28/swagger/servicefabric.json

```diff
@@ -1129,6 +1129,13 @@
         ],
         "parameters": [
           {
+            "name": "timeout",
+            "in": "query",
+            "description": "The timeout in seconds",
+            "type": "integer",
+            "required": false
```

Comment:
What happens if no timeout is specified? If there is a default value, you may document it.